modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/exportimport/portlet/preferences/processor/IGDisplayExportImportPortletPreferencesProcessor.java

```diff
@@ -79,8 +79,10 @@ public PortletPreferences processImportPortletPreferences(
 				portletDataContext, portletPreferences);
 	}
 
-	@Reference
-	private DLExportImportPortletPreferencesProcessor
+	@Reference(
+		target = "(component.name=com.liferay.document.library.web.internal.exportimport.portlet.preferences.processor.DLExportImportPortletPreferencesProcessor)"
```

Comment:
@RenanUmV, please check if it is possible to filter by the `javax.portlet.name` property. Thx